tools/bin/ariaAOIassist.py

```diff
@@ -1,371 +1,505 @@
-#!/usr/bin/env python3
+#!/anaconda3/envs/ARIA-tools/bin/python3
```

Comment:
this is local to your machine

Reply:
Right I noticed that, but it was written /anaconda3/... in the original code, so I must have used an older version of it. Good fix, thanks.

Reply:
can you correct it to what it used to be?

Reply:
Yes, but I look at the way ariaAOIassist is currently written is the standard way, i.e., !/usr/bin/env python3. I should keep that, right?

Reply:
yes that woudl be consistent with the other programs:
```
#!/usr/bin/env python3
```